[50955] Rename time fields in the UI

cbliard opened the pull request:
https://community.openproject.org/wp/50955

Estimated time => Work
Remaining time => Remaining work
Progress (%) => % Complete

oliverguenther commented:
This is scoped for 13.2., correct? So to merge this, we'll need to create a release branch first.

cbliard commented:
Yes it is. I'll mark it as draft.

cbliard commented:
But sadly draft PR cannot be reviewed.

cbliard commented:
It's only a flaky test which is failing.

cbliard committed:
[50955] Rename time fields in the UI

Estimated time => Work
Remaining time => Remaining work
Progress (%) => % Complete

The export pdf test needed to be fixed too: The test has been updated to expect
only 1 page instead of two. As fields having been renamed, they are now
shorter. So some vertical space is saved and the additional page break
is not triggered anymore before rendering the custom fields.

ulferts commented on spec/features/work_packages/select/select_query_spec.rb:
```diff
@@ -83,7 +83,7 @@
       filters.open
       filters.expect_filter_by 'Assignee', 'is (OR)', ['me']
-      filters.expect_filter_by 'Progress (%)', '>=', ['10'], 'percentageDone'
+      filters.expect_filter_by 'Percent Complete', '>=', ['10'], 'percentageDone'
```
I believe this should be '% Complete'

cbliard replied:
Actually, it does not really matter. In `expect_filter_by`, the name is not used if there is a selector. I was surprised that it was still passing after renaming, so I changed it to something which is not meaningful (does not match selector or translated name) to make it a bit more self-explanatory that this parameter is actually not relevant.

ulferts reviewed (changes requested):
Only one teeny tiny thing, @cbliard. Feel free to merge yourself after addressing.
